cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,9 +25,11 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
+
 
 def events_check(updates=False):
-    form_bug = BZ(1485953, forced_streams=['5.7', '5.8', 'upstream'])
+    form_bug = BZ(1485953, forced_streams=['upstream'])
```

Comment:
Does this not apply to 5.9 or 5.10? Just upstream that is does not work?

Reply:
@jawatts @digitronik 
Please help me with this comment.

- I am not sure if this is exist on ```5.10 or 5.9```.
- I went through BZ but I am not getting clear idea behind this BZ.
- As it was reported for ```5.7, 5.8```, also according to [this comment](https://bugzilla.redhat.com/show_bug.cgi?id=1485953#c9); this BZ does not exist on 5.10 or 5.9.
- For more confirmation, I would like to ask @mshriver about it; as it was reported by Mike.
- For now I am updating with removing this ``` form_bug```.

Thank you

Reply:
I think you should remove `upstream` as well.

Reply:
@digitronik agree. Looks like @ganeshhubale did remove it in the lastest.